### ISSUE TITLE
alignment spikes

### ISSUE THREAD

**Issue by notconvergingwtf** [deepinsight/insightface#572]
Hi, do you have any suggestions on the next problem about facial landmarks:
While training sdu(nadam,lr=0.00025), this is the loss on validation test:
![image](https://user-images.githubusercontent.com/48060406/53564775-d2c0d080-3b68-11e9-8c6e-5c2cda496ac0.png)
Different model on the same training data was fine
Also, while training, lossvalue=nan starts to appear from epoch,depended on lr

**Comment by nttstar** [comment]
Are you training on 3d dataset?

**Comment by deepinx** [comment]
@nttstar I also meet the same situation on both 2d and 3d datasets, can you tell what's the problem. Thanks.

**Comment by notconvergingwtf** [comment]
@nttstat i’ve used 2d one

**Comment by nttstar** [comment]
Hi, I have not met this issue on 2d dataset.  For 3d dataset, recently I just train it for about 14K iterations.

**Comment by nttstar** [comment]
Can you firstly check if hourglass/HPM networks can work well?

**Comment by notconvergingwtf** [comment]
@nttstar Seems that dividing lr by 10^3 solves the problem (lr~10^-6) for both nets. Thanks
